src/zoid/buttons/container.jsx

```diff
@@ -53,12 +54,13 @@ export function containerTemplate({ uid, props, tag, context, frame, prerenderFr
     });
 
     // $FlowFixMe
-    const { style } = props;
+    const { style, csp, nonce: nonceProp } = props;
```

Comment:
My memory is fuzzy on this -- but is it actually possible to pass `nonce` as a raw prop right now?

If so, would it make sense to just define this as a prop in buttons/component.jsx and default it there?

```
nonce: {
    type: 'string',
    default: getCSPNonce
}
```

Reply:
Thanks @bluepnume that's a great suggestion. We do have `nonce` defined as a prop right now (https://github.com/paypal/paypal-checkout-components/blob/master/src/ui/buttons/props.js#L320). I made this change and got rid of the normalizeNonce helper function.

Do you think we should remove the `csp` prop? https://github.com/paypal/paypal-checkout-components/blob/master/src/zoid/buttons/component.jsx#L438-L446. With this approach I don't think it's needed anymore. We can always leverage `getCSPNonce()` from sdk-client when we need to know if its a global sdk script nonce vs a nonce passed in to a component as a prop. 🤔

Reply:
I did not remove the `csp` prop since it seems valuable to know the source of the nonce.